forms-flow-nav/src/Navbar.jsx

```diff
@@ -139,6 +141,14 @@ const NavBar = React.memo(({ props }) => {
     }
   }, [instance]);
 
+  React.useEffect(() => {
+    if (location.pathname.includes('public') && !isAuthenticated) {
+      setApplicationTitle("formsflow.ai");
```

Comment:
Don’t we have a generic logo change page ?

Reply:
Also application title comes from env for cases other than multitenancy

Reply:
I think if the form name is public this useEffect will work. please check that